scripts/homebrew/pr_template.md

```diff
@@ -0,0 +1,3 @@
+## Description
+
+A new release has been published on {{server_url}}/{{repository}}: [`{{repository}} {{tag}}`]({{server_url}}/{{repository}}/actions/runs/{{run_id}})
```

Comment:
Does the release page already exist at this point? It would be more interesting than the workflow run

Reply:
Yeah, I agree, and I changed it.